features/e2e_tests/performance_test.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+function local_env() {
+    BASEDIR="$(dirname "$0")"
+    source "${BASEDIR}/../lib/functions.sh"
+}
+
+function validate_function_by_worker() {
```

Comment:
Extract the duplicated code to a _library_ (like the `lib/functions.sh`)

Reply:
I cannot do that in this case, I tried it but the command it's executed when the script it's parsed and I need something like a lazy execution because of a `${worker}` var still not exists at this point.